data-extraction/pyDataExtraction/__main__.py

```diff
@@ -0,0 +1,10 @@
+from json import dumps
+from typing import Union, Dict, Optional
+from abc import ABC, abstractmethod
+from pyDataExtraction.commonTypes.Graph import Graph
+
+if __name__ == "__main__":
+    graph_data1 = {"A": ["B", "C"], "B": ["A,C"], "C": ["A,D"], "D": ["A"]}
```

Comment:
I think "A,C" and "A,D" be separated?
```
graph_data1 = {"A": ["B", "C"], "B": ["A", "C"], "C": ["A", "D"], "D": ["A"]}
```

Reply:
Multiline formnatting of dictionary seems to be more readable

Reply:
I've mainly be using black for formatting, and it seems to condense it down if a line is less than 88 characters.

also, the main is current just representing how this would be used python side, but this will be where typescript interfaces with the code, passing information and returning information to the typescript for visualization, so this is going to be reworked extensively once I can get a look at the data format passed from the `PyEvaluationEngine.ts` file.